reference/halyard/custom.md

```diff
@@ -116,3 +116,7 @@ kubernetes:
     example/annotation: spinnaker.io
     example/annotation-2: halyard
 ```
+
+#### useExecHealthCheck
```

Comment:
@paulczar 
I was thinking move it to theend of the table, not the end of the doc. Previously, you had this in the table of "tweakable service settings." Now it's under Kubernetes-specific settings. If this setting is more appropriate for K8s-specifc settings, then yes this is a good place for it.

Reply:
thanks @dorbin,  we did move it into the kubernetes-specific settings, so this is indeed the right place for it. 

see https://github.com/spinnaker/halyard/pull/1205